src/arrays.js

```diff
@@ -1,61 +1,66 @@
 const getNthElement = (index, array) => {
-  // your code here
+index %= array.length
+  return array[index];
+
 };
 
 const arrayToCSVString = array => {
-  // your code here
+return array.join();
 };
 
 const csvStringToArray = string => {
-  // your code here
+return string.split (',');
```

Comment:
return statement missing indentation, and there is n extra space after split before (), which should be removed